stdlib/Distributed/src/cluster.jl

```diff
@@ -720,13 +720,36 @@ const map_del_wrkr = Set{Int}()
     myid()
 
 Get the id of the current process.
+
+# Examples
+```julia-repl
+julia> myid()
+1
+
+julia> remotecall_fetch(() -> myid(), 4)
+4
+```
 """
 myid() = LPROC.id
 
 """
     nprocs()
 
 Get the number of available processes.
+
+# Examples
+```julia-repl
+julia> nprocs()
+6
+
+julia> workers()
+5-element Array{Int64,1}:
```

Comment:
Perhaps showing 3 workers is enough to get the point across? Saves some vertical space :)